warp/service/warp_service.go

```diff
@@ -0,0 +1,35 @@
+// (c) 2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package service
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+)
+
+type SignatureResponse struct {
+	Signature hexutil.Bytes
+}
+
+// WarpAPI introduces snowman specific functionality to the evm
+type WarpAPI struct {
+	Backend WarpBackend
+}
+
+// GetSignature returns the BLS signature associated with a messageID. In the raw request, [messageID] should be cb58 encoded
+func (api *WarpAPI) GetSignature(ctx context.Context, messageID ids.ID) (*SignatureResponse, error) {
+	signature, err := api.Backend.GetSignature(ctx, messageID)
```

Comment:
We should get rid of `SignatureResponse` type and just return `[]byte` . If we wrap the `[]byte` type internally with the type `hexutil.Bytes` it should marshal it to a correct hex output instead of returning byte by byte.